client-runtime/src/test/java/com/microsoft/rest/UserAgentTests.java

```diff
@@ -15,52 +22,57 @@
 import okhttp3.Interceptor;
 import okhttp3.OkHttpClient;
 import okhttp3.Protocol;
-import okhttp3.Request;
 import okhttp3.Response;
-import retrofit2.Retrofit;
+import retrofit2.http.HTTP;
```

Comment:
Didn't quite get rid of retrofit2?

Reply:
Whoops, will take another look at that one.